<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ligar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="trilho">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>561</width>
      <height>411</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>561</width>
      <height>411</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/windows-mint/sprint-trilho.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(222, 222, 222)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #fff}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title_header">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(221, 221, 221)}</string>
    </property>
    <property name="text">
     <string>THREAD TRAIN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="velocity_background">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>161</width>
      <height>185</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sawasdee</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: rgb(221, 221, 221)}</string>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QSlider" name="slider_t1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="color_t1">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>215</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background:#406BFF}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="color_t2">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>245</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background:#5C6999}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider_t2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_t3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>265</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="color_t3">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>270</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background:#33CCFF}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="slider_t4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="color_t4">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>295</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background:#FFA673}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="slider_t5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>315</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="color_t5">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>320</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background:#CC3C14}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trem3">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/windows-mint/sprite-train-down.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_trem2</zorder>
   <zorder>label_trem1</zorder>
   <zorder>background</zorder>
   <zorder>trilho</zorder>
   <zorder>sidebar</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>title_header</zorder>
   <zorder>velocity_background</zorder>
   <zorder>slider_t1</zorder>
   <zorder>color_t1</zorder>
   <zorder>color_t2</zorder>
   <zorder>slider_t2</zorder>
   <zorder>slider_t3</zorder>
   <zorder>color_t3</zorder>
   <zorder>slider_t4</zorder>
   <zorder>color_t4</zorder>
   <zorder>slider_t5</zorder>
   <zorder>color_t5</zorder>
   <zorder>trem3</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
